libs/clustermatch/plots.py

```diff
@@ -146,38 +144,28 @@ def plot_cumulative_histogram(
         coef_at_percent = data.quantile(gene_pairs_percent)
 
         # show values so it is saved in the notebook
-        display(coef_at_percent)
+        display(coef_at_percent.sort_values())
 
         x_lim = ax.get_xlim()
+        hline_x_max = coef_at_percent.max()
+
         ax.hlines(
             y=gene_pairs_percent * 100,
             xmin=x_lim[0],
-            xmax=coef_at_percent["spearman"],
-            color="gray",
-            linestyle="dotted",
-        )
-        ax.vlines(
-            x=coef_at_percent["clustermatch"],
-            ymin=0,
-            ymax=gene_pairs_percent * 100,
-            color="gray",
-            linestyle="dotted",
-        )
-        ax.vlines(
-            x=coef_at_percent["pearson"],
-            ymin=0,
-            ymax=gene_pairs_percent * 100,
-            color="gray",
-            linestyle="dotted",
-        )
-        ax.vlines(
-            x=coef_at_percent["spearman"],
-            ymin=0,
-            ymax=gene_pairs_percent * 100,
+            xmax=hline_x_max,
             color="gray",
             linestyle="dotted",
         )
 
+        for method_name in coef_at_percent.index:
```

Comment:
This looks a lot nicer than the previous iteration! It inspires me to make my code nicer hahaha